### DISCUSSION TITLE
SiteName in Google search results problem

### DISCUSSION THREAD

**Discussion by devmilek** [vercel/next.js#65793] [Help]
### Summary

<img width="628" alt="Zrzut ekranu 2024-05-15 o 15 50 22" src="https://github.com/vercel/next.js/assets/61127609/af81ed2b-6efa-4a35-8c4a-8580bfde21a0">

Why doesn't my site's icon and SiteName appear in google results, I tried reindexing the site in Google Search Console, I also added Ld+Json Schema and it still doesn't work.

<img width="328" alt="Zrzut ekranu 2024-05-15 o 15 51 58" src="https://github.com/vercel/next.js/assets/61127609/ae99cf9c-3782-44f1-9623-b0ba1c6ee4aa">
<img width="528" alt="Zrzut ekranu 2024-05-15 o 15 52 19" src="https://github.com/vercel/next.js/assets/61127609/78be2da9-23c3-4940-a8ba-1fe8af7b6a2e">

Favicon is correctly sized at 48x48px and added in root directory.

Please help 😑

**Comment by victornasar** [comment]
Im having he same issues. I thought my favicon.ico in the app directory would've been enough. I'm just left with a generic globe on google search

**Reply by Strong-Ari** [reply]
I have also the same problem

**Comment by aleksa-codes** [comment]
any update on this? I am facing the same issue, and tried almost everything, but the site name still shows the URL in Google search results.

**Comment by bdobruno** [comment]
Me too

**Reply by immiProgrammer** [reply]
try to add favicon.ico in public dir and correctly link in metadata

**Reply by Strong-Ari** [reply]
Don't worry. If you've placed the icon in the right spot, it will appear soon. This is a common Vercel error that doesn't seem to affect everyone. Also, carefully check the documentation and don't overthink it; just follow their instructions. [https://nextjs.org/docs/app/api-reference/file-conventions/metadata/app-icons](url)
